apps/core/lib/core/snitch/data/model/order.ex

```diff
@@ -20,14 +20,14 @@ defmodule Core.Snitch.Data.Model.Order do
   end
 
   @spec delete(non_neg_integer | Schema.Order.t()) ::
-          {:ok, Schema.Order.t()} | {:error, Ecto.Changeset.t()}
+          {:ok, Schema.Order.t()} | {:error, Ecto.Changeset.t()} | {:error, :not_found}
   def delete(id_or_instance) do
     QH.delete(Schema.Order, id_or_instance, Repo)
   end
 
-  @spec get(map) :: Schema.Order.t() | nil | no_return
-  def get(query_fields) do
-    QH.get(Schema.Order, query_fields, Repo)
+  @spec get(map | non_neg_integer) :: Schema.Order.t() | nil | no_return
+  def get(query_fields_or_primary_key) do
```

Comment:
why was there a need for change in the name, I guess `query_fields` was self explanatory it can also contain a primary key.
IMO here the name kinda of states that there will be a `query_fields`(i.e a map) or a `primary_key`(i.e a integer(a singular entity)).
Please let me know your thoughts on this. 😄 
cc: @ashish173  @pkrawat1

Reply:
> Here the name kinda of states that there will be a query_fields(i.e a map) or a primary_key(i.e a integer(a singular entity)).

That is exactly what it is. Either a `map`, or an `integer`. See the spec on line 28:
`@spec get(map | non_neg_integer) :: Schema.Order.t() | nil | no_return`